src/lib.rs

```diff
@@ -76,7 +79,8 @@ pub use self::error::{ErrorReason, Error};
 pub use self::pod::{PodTransmutable, guarded_transmute_pod_many_permissive, guarded_transmute_pod_vec_permissive, guarded_transmute_pod_many_pedantic,
                     guarded_transmute_pod_pedantic, guarded_transmute_pod_vec_pedantic, guarded_transmute_pod_many, guarded_transmute_pod,
                     guarded_transmute_pod_vec};
-
+pub use self::bool::{guarded_transmute_bool_permissive, guarded_transmute_bool_pedantic,
```

Comment:
Are these sorted from longest to shortest?

Reply:
Is it ok now?